modules/install-ibm-cloud-configuring-the-public-subnet.adoc

```diff
@@ -0,0 +1,160 @@
+// This is included in the following assemblies:
+//
+// installing_ibm_cloud/install-ibm-cloud-installing-on-ibm-cloud.adoc
+
+[id="configuring-the-public-subnet_{context}"]
+= Configuring the public subnet
+
+All of the {product-title} cluster nodes must be on the public subnet. IBM Cloud&#174; does not provide a DHCP server on the subnet. Set it up separately on the provisioner node. This document describes configuring Dnsmasq on the public subnet.
+
+.Procedure
+
+. Ensure `dnsmasq` is running:
++
+[source,terminal]
+----
+$ systemctl status dnsmasq
+----
++
+If `dnsmasq` isn't running, start it and enable it:
++
+[source,terminal]
+----
+$ sudo systemctl start dnsmasq
+----
++
+[source,terminal]
+----
+$ sudo systemctl enable dnsmasq
+----
+
+. Set an unused IP address from the public IP range:
++
+[source,terminal]
+----
+$ UNUSEDIP=<IP-address>
+----
+
+. Make a directory for the Dnsmasq configuration files:
++
+[source,terminal]
+----
+$ sudo mkdir -p  /var/lib/baremetal-dhcp
+----
+
```

Comment:
No longer needed, going to use /etc/dnsmasq.conf instead

Reply:
done